src/server/services/gitHubService/reactivateUser.js

```diff
@@ -0,0 +1,8 @@
+import {Chapter} from 'src/server/services/dataService'
+import addUserToTeam from './addUserToTeam'
+
+export default async function reactivateUser(user) {
+  const chapter = await Chapter.get(user.chapterId)
```

Comment:
Let's delete this function and directly call `addUserToTeam()` from the `reactivateUser`  server action.

The primary reason is that we don't want to create a direct dependency of the `githubService` on the `dataService` -- they shouldn't know anything about each other. Use the dataService in the `reactivateUser` server action to get the team ID and pass that into `githubService.addUserToTeam`.